code/modules/projectiles/projectile/bullets.dm

```diff
@@ -429,6 +429,13 @@
 	if(istype(atarget, /mob/living) && damage == 200)
 		var/mob/living/M = atarget
 		M.gib()
+	else if(istype(atarget, /obj/machinery/singularity/narsie) && blessed && damage == 200) //MINE IS THE ROD THAT SHALL PIERCE THE HEAVENS
+		var/obj/machinery/singularity/narsie/N = atarget
+		if(!N.wounded)
+			N.visible_message("<span class = 'danger'>\The [src] strikes \the [N], wounding them. This god can bleed!</span>", range = 20)
```

Comment:
I'm a bit disappointed this is all that happens for a setup that is probably one in a million to happen.

Reply:
Well, depends on how many times narsie is shot overall over multiple rounds. Deity didn't want narsie being absolutely destroyed as he's a 'constant' between rounds.

We'll see for the future.

Reply:
You don't have to destroy him, you could banish him back or weaken the veil causing all the summoned monsters to go away.

Reply:
That'd be crew victory. You don't banish an elder god with a railgun, it's less about reversing the tide and more about bragging rights for having managed to harm Nar-Sie. The round is over anyway at that point.

Reply:
>You don't banish an elder god with a railgun

b-but to defeat the Cyberdemon...